test/test_vmap.py

```diff
@@ -3430,11 +3422,13 @@ def reset_random():
         generator = torch.Generator(device=device)
         orig_state = generator.get_state()
         kwargs = {'device': device, 'generator': generator} if use_generator else {'device': device}
+        only_gen_kwarg = {'generator': generator} if use_generator else {}
         supported_random_ops = [
             lambda _, shape: torch.randn(shape, **kwargs),
             lambda _, shape: torch.rand(shape, **kwargs),
             lambda _, shape: torch.randint(100, shape, **kwargs),
             lambda _, shape: torch.randint(5, 100, shape, **kwargs),
+            lambda t, _: t.random_(**only_gen_kwarg),
```

Comment:
(observation, no action required) ack, it's a little weird that this function takes in two args and the others take in a single arg